### DISCUSSION TITLE
NHIN for CTEN

### DISCUSSION THREAD

**Discussion by coasthie** [nextgenhealthcare/connect#5661] [Q&A]
The organization I work for is joining CTEN and will use NHIN web services specification to exchange messages with partner organizations.

Are there any open source Mirth extensions for implementing any of the following from https://www.ca-hie.org/initiatives/cten/specifications/

    Nationwide Health Information Network (NHIN) Patient Discovery Web Service Interface Specification v2.0
    Nationwide Health Information Network (NHIN) Query for Documents Web Service Interface Specification v3.0
    Nationwide Health Information Network (NHIN) Retrieve Documents Web Service Interface Specification v3.0

**Comment by ralphb918** [comment]
Hi coasthie,

With regard to an open source software product for connecting to the national data networks, there used to be a program called CONNECT which hasn't been supported since 2019. We haven't come across any other open source options. 

**CONNECT Wiki Support Ended on September 6, 2019
The Federal Health Architecture (FHA) and the Office of the National Coordinator for Health It (ONC) are ending support for the CONNECT open source software project in September 2019. As part of ending the support for CONNECT, the FHA and ONC will no longer maintain or update the CONNECT Wiki after September 6, 2019 at 5:00 PM EDT. 

As an open source project, the CONNECT software code and community resources, including this Wiki, can be used, adopted and implemented by any interested organization, but the government will no longer be involved and therefore will not endorse or verify the quality of the resources previously managed by the CONNECT Community under FHA’s guidance.**

At Zen Healthcare IT we have solutions for the types of data exchanges you are looking for and we are a Carequality implementer. We offer this as a SAAS offering so not free or open source, but if you find that there are no other options for your team please drop me a note at ralphb@consultzen.com and I can get you some additional information.  Thanks

**Reply by twest-mirthconnect** [reply]
@coasthie - Feel free to give me a call or DM. We are happy to help with any of your Mirth Connect questions.

Thanks,
Travis West

Mirth Connect Product Management
615-556-4202
twest@nextgen.com
